### PR TITLE
Practice pull request

### PR DESCRIPTION
**Pull request by Lightining1boy**
Started making database tables

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+DROP DATABASE IF EXCISTS reviews
```
Comment (sbtorres) [inline schema.sql]:
```suggestion
DROP DATABASE IF EXISTS reviews
```
Small typo on EXISTS statement

File: schema.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+CREATE DATABASE reviews
+
+USE reviews
+
+CREATE TABLE Messages (
```
Comment (sbtorres) [inline schema.sql]:
```suggestion
CREATE TABLE messages (
```
Convention is usually to use all lower case for MySQL, up to you if you want to have it capitalized though.

File: schema.sql [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  id int NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  User_id int NOT NULL FOREIGN KEY,
```
Comment (sbtorres) [inline schema.sql]:
```suggestion
  user_id int NOT NULL FOREIGN KEY,
```
Same thing here with capitalization convention

File: schema.sql (hunk 1 above)
Comment (Lightining1boy) [reply schema.sql]: Thanks Scott i'll go ahead and fix it!

File: schema.sql (hunk 2 above)
Comment (Lightining1boy) [reply schema.sql]: Better to stick with what is conventional, i'll go ahead and change it.

File: schema.sql (hunk 3 above)
Comment (michelledhuang23) [reply schema.sql]:
```suggestion
  user_id INT NOT NULL FOREIGN KEY,
```

**Pull request merged**